### DISCUSSION TITLE
No option to close Select upon choice

### DISCUSSION THREAD

**Discussion by mihey278** [ant-design/ant-design#30205] [Q&A]
Currently, the `Select` (tag / multiple mode) API does not offer the ability to close the select dropdown when a user chooses an option. This is creating a negative user experience (example: https://quantisnow.com/feed), especially on mobile (hard to find a place outside the dropdown to press for closing without pressing another button mistakenly)

How can this be achieved given:
- Functional component
- Typecript

Thanks.

**Comment by huntdream** [comment]
You can render a [custom dropdown](https://ant.design/components/select/#components-select-demo-custom-dropdown-menu) with a close button
